openexp/_canvas/canvas.py

```diff
@@ -1194,8 +1194,12 @@ def _gabor(orient, freq, env=u"gaussian", size=96, stdev=12, phase=0,
 			# Apply the envelope
 			if bgmode == u"avg":
 				amp = amp * f + 0.5 * (1.0 - f)
-			else:
+			elif bgmode == u"col2":
 				amp = amp * f
+			else:
+				raise osexception((u"Invalid argument for bgmode: %s " % bgmode) + 
```

Comment:
This kind of hurts my eyes and my brain. Would you mind making it a bit cleaner? (Free protip: You don't need a `+` to concatenate strings: `'x' 'y' == 'xy'`)

~~~ .python
raise osexception(u"Invalid argument for bgmode: %s (should be one of 'avg','col2')" % bgmode)
~~~